plugins/source/aws/codegen/recipes/ssoadmin.go

```diff
@@ -19,6 +21,13 @@ func SSOAdminResources() []*Resource {
 			Struct:              &types.PermissionSet{},
 			Description:         "https://docs.aws.amazon.com/singlesignon/latest/APIReference/API_PermissionSet.html",
 			PreResourceResolver: "getSsoadminPermissionSet",
+			ExtraColumns: []codegen.ColumnDefinition{
+				{
+					Name:     "inline_policy",
+					Type:     schema.TypeString,
```

Comment:
If i recall correctly this polict is actually valid JSON... so I think the type should be `schema.TypeJSON`